Wikipedia/Code/WMFReferencePageBackgroundView.swift

```diff
@@ -0,0 +1,21 @@
+
+class WMFReferencePageBackgroundView: UIView {
+    var clearRect:CGRect = CGRectZero {
+        didSet {
+            setNeedsDisplay()
+        }
+    }
+
+    override func drawRect(rect: CGRect) {
```

Comment:
Not a big deal, but any reason you went with draw rect instead of just setting the corner radius on the self.layer?